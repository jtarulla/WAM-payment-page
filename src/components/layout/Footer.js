import React from 'react';
import logo from '../../assets/logo.jpg';

export const Footer = () => {
	return (
		<footer className='footer' data-testid='footer'>
			<>
				<div className='copyright'>2020 &copy; We Are Marketing</div>

				<div className='logo'>
					<a
						href='https://www.wearemarketing.com/es/'
						target='_blank'
						rel='noopener noreferrer'
					>
						<img src={logo} alt='WAM' />
					</a>
				</div>
			</>
		</footer>
	);
};
