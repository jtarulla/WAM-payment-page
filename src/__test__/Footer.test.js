import { shallow } from 'enzyme';
import React from 'react';
import Footer from '../components/layout/Footer';

it('render Footer component', () => {
	expect(shallow(<Footer />)).toMatchSnapshot();
});
