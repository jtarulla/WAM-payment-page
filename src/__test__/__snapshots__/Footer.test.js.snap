// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render Footer component 1`] = `ShallowWrapper {}`;
